app/assets/stylesheets/components/letter_preview.scss

```
@@ -0,0 +1,47 @@
+.letter_preview {
+  border: 2px solid $hackney-green;
+  padding: 20px;
+  max-height: 50vh;
+  overflow: scroll;
+
+  .right{
```

currently, the frontend is using the BEM naming scheme. We should stay consistent to that.

So in this example we do
```suggestion
  .&--right{
```
that creates `.letter_preview--right`